set.mm

```diff
@@ -47773,12 +47834,14 @@ although the definition does not require it (see ~ dfid2 for a case
   $}
 
   ${
-    $d x ph $.  $d y ph $.
+    $d ph x z $.  $d ph y z $.  $d ps z $.  $d ch z $.
     opabbidv.1 $e |- ( ph -> ( ps <-> ch ) ) $.
     $( Equivalent wff's yield equal ordered-pair class abstractions (deduction
        form).  (Contributed by NM, 15-May-1995.) $)
     opabbidv $p |- ( ph -> { <. x , y >. | ps } = { <. x , y >. | ch } ) $=
-      ( nfv opabbid ) ABCDEADGAEGFH $.
+      ( vz cv cop wceq wa wex cab copab anbi2d 2exbidv abbidv df-opab 3eqtr4g )
+      AGHDHEHIJZBKZELDLZGMTCKZELDLZGMBDENCDENAUBUDGAUAUCDEABCTFOPQBDEGRCDEGRS
+      $.
```

Comment:
avoided axioms?

Reply:
it saves ax-12

Reply:
Yes, as @icecream17 wrote in a previous MR, it's almost an automatic change: instead of using nfv, use the proof of oppabid with the *v versions. Therefore, it does not require a revision tag, in @icecream17's and my opinions.

Reply:
I thought about a tag "$( $j usage 'opabbidv' avoids 'ax-12' ...", but this is non-blocking.

Reply:
You're right @avekens, sorry I overlooked that.  @icecream17 added it, so I'm resolving the conversation, and merging.